### PR TITLE
[dev-v2.10] rancher-cis-benchmark auto-bump 105.0.1

### PR DESCRIPTION
**Pull request by nicholasSUSE**
Auto bumping rancher-cis-benchmark

### CODE REVIEW

File: release.yaml [hunk 1]
```diff
@@ -23,8 +23,10 @@ neuvector-monitor:
   - 105.0.0+up2.8.3
 rancher-cis-benchmark:
   - 105.0.0+up7.0.0
+  - 105.0.1+up7.0.1
 rancher-cis-benchmark-crd:
   - 105.0.0+up7.0.0
```
Comment (mitulshah-suse) [inline release.yaml:27]: @nicholasSUSE this version is already released right? shouldnt it be removed already?

Comment (nicholasSUSE) [reply release.yaml:27]: Yes it should be removed but it does not break anything. 
I will be removing it.

Comment (nicholasSUSE) [reply release.yaml:27]: Done and thanks for reviewing!

**Review by mitulshah-suse — APPROVED** [review]
LGTM

**Pull request merged**